qa/workunits/cephtool/test.sh

```diff
@@ -2343,11 +2343,15 @@ function test_mon_deprecated_commands()
 
 function test_mon_cephdf_commands()
 {
+  # before creating lots of PG, reweight the OSDs. there is chance
+  # that crush fails to pick an OSD for a PG due to overload
+  ceph osd reweight-by-pg 110
+
   # ceph df detail:
   # pool section:
   # RAW USED The near raw used per pool in raw total
 
-  ceph osd pool create cephdf_for_test 32 32 replicated
+  ceph osd pool create cephdf_for_test 16 16 replicated
```

Comment:
how about just 1 instead of 16?

also, should we add a wait_for_health_ok in there?

Reply:
if we have pg=pgp=1, i don't think reweight-by-pg is necessary anymore, hence no need to wait_for_health_ok.